packages/trader/src/AppV2/Utils/trade-types-utils.tsx

```diff
@@ -25,6 +25,21 @@ export const CONTRACT_LIST = {
     EVEN_ODD: 'Even/Odd',
     OVER_UNDER: 'Over/Under',
 };
+export const getContractTypesList = () => ({
```

Comment:
Why do we have/need so many mappers? 😄

Reply:
Agree this is getting too much lol. Theres gotta be a better way around this

Reply:
There's already `getContractTypesList` function in useActiveSymbols hook. It would be nice to give it a different name to avoid confusions, plus it is doing the same thing

Reply:
Sorry Akmal, I didn't notice that we have function with the same name. No, it's not doing the same: your function is based on `getContractTypesConfig` and returns trade types like this: 'CALL', 'DIGITEVEN'. My is returning 'Rise/Fall', 'Even/Odd'

Reply:
CONTRACT_LIST returns human-readable contract name, TRADE_TYPES - returns contract's short name, how we are calling them in trade store.
Example: TRADE_TYPES.OVER_UNDER = 'over_under' VS CONTRACT_LIST.OVER_UNDER = 'Over/Under'

Reply:
I will find a better way for handling this, thank you